<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bitplane</class>
 <widget class="QWidget" name="bitplane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>1121</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="bit1">
        <property name="text">
         <string>bit_1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bit2">
        <property name="text">
         <string>bit_2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bit3">
        <property name="text">
         <string>bit_3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bit4">
        <property name="text">
         <string>bit_4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="bit5">
        <property name="text">
         <string>bit_5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bit6">
        <property name="text">
         <string>bit_6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bit7">
        <property name="text">
         <string>bit_7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bit8">
        <property name="text">
         <string>bit_8</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
